### ISSUE TITLE
cannot build powerpc64-linux-gnu-gcc due to conflict

### ISSUE THREAD

**Issue by goekce** [actionless/pikaur#377]
```
pikaur -S powerpc64-linux-gnu-gcc
```

##### Attached log:

```
Reading repository package databases...
Reading local package database...
Resolving AUR dependencies...

:: New dependencies will be installed from repository:
 dejagnu (for powerpc64-linux-gnu-gcc, powerpc64-linux-gnu-binutils)    -> 1.6.2-1
 expect (for dejagnu)                                       -> 5.45.4-1
 gperf (for powerpc64-linux-gnu-glibc)                      -> 3.1-2
 tcl (for expect)                                           -> 8.6.9-2

:: AUR package will be installed:
 powerpc64-linux-gnu-gcc                                    -> 9.1.0-1

:: New dependencies will be installed from AUR:
 powerpc64-linux-gnu-binutils (for powerpc64-linux-gnu-gcc, powerpc64-linux-gnu-gcc-stage1, powerpc64-linux-gnu-gcc-stage2)     -> 2.32-2
 powerpc64-linux-gnu-gcc-stage1 (for powerpc64-linux-gnu-glibc-headers)     -> 9.1.0-4
 powerpc64-linux-gnu-gcc-stage2 (for powerpc64-linux-gnu-glibc-headers, powerpc64-linux-gnu-glibc)     -> 9.1.0-4
 powerpc64-linux-gnu-glibc (for powerpc64-linux-gnu-gcc, powerpc64-linux-gnu-gcc-stage2)     -> 2.29-1
 powerpc64-linux-gnu-glibc-headers (for powerpc64-linux-gnu-gcc-stage2)     -> 2.29-1
 powerpc64-linux-gnu-linux-api-headers (for powerpc64-linux-gnu-glibc-headers, powerpc64-linux-gnu-glibc)     -> 5.1-1

:: Proceed with installation? [Y/n] 
:: [v]iew package details   [m]anually select packages
>> 
looking for conflicting AUR packages...
New packages 'powerpc64-linux-gnu-gcc' and 'powerpc64-linux-gnu-gcc-stage2' are in conflict.

```
##### Workaround:
```
pikaur -S powerpc64-linux-gnu-binutils
pikaur -S powerpc64-linux-gnu-glibc-headers
pikaur -S powerpc64-linux-gnu-gcc-stage2
pikaur -S powerpc64-linux-gnu-gcc

**Comment by actionless** [comment]
```
$ pikaur -Si powerpc64-linux-gnu-gcc powerpc64-linux-gnu-gcc-stage2 | grep -E '^(Name|Conflicts)'
Name                      : powerpc64-linux-gnu-gcc
Conflicts With            : powerpc64-linux-gnu-gcc-stage1, powerpc64-linux-gnu-gcc-stage2
Name                      : powerpc64-linux-gnu-gcc-stage2
Conflicts With            : powerpc64-linux-gnu-gcc-stage1
```

**Comment by goekce** [comment]
@actionless You mean that the dependency should not be described like this?

I contacted the package maintainer and they confirmed that "yay" also cannot solve the dependencies. After their suggestion a patch was implemented for yay which solved the dependency problem, but introduced other bugs. This led to the removal of the patch.

According to the maintainer the dependency is as follows:
1. powerpc64-linux-gnu-binutils
2. powerpc64-linux-gnu-gcc-stage1
3. powerpc64-linux-gnu-linux-api-headers
4. powerpc64-linux-gnu-glibc-headers
5. powerpc64-linux-gnu-gcc-stage2 (ersetzt gcc-stage1)
6. powerpc64-linux-gnu-glibc (ersetzt glibc-headers)
7. powerpc64-linux-gnu-gcc (ersetzt gcc-stage2)

Here is additional info for building cross-compiling gcc:
https://git.parabola.nu/abslibre.git/tree/cross/cross-gcc/PKGBUILD
